hbt/selection/jet.py

```diff
@@ -56,6 +56,12 @@ def jet_selection(
     https://twiki.cern.ch/twiki/bin/view/CMS/PileupJetIDUL?rev=17
     https://twiki.cern.ch/twiki/bin/view/CMSPublic/WorkBookNanoAOD?rev=100#Jets
     """
+    # remove coffea columns only used for internal processes. This hinder ellipses slicing due to dimensionality issues
+    # local_index is also affected by this behavior
+    for jet_route in get_ak_routes(events.Jet):
+        if jet_route.column.endswith("IdxG"):
+            events = remove_ak_column(events, f"Jet.{jet_route}")
```

Comment:
Check if we can move from ellipses to normal, dimension-defining slices instead.